analysis_plan.md

```diff
@@ -0,0 +1,20 @@
+# First analysis plan
+
+This is the first analysis plan.
+
+It is provisional.
+
+## Reading to do
+
+* Read about Nipy diagnostics
+* Consider Principal Component Analysis (good tutorial?)
+* Consider Independent Component Analysis (e.g. FSL)
+* Your ideas here.
```

Comment:
I think the Gabrieli lab had some tool about a decade ago...

Reply:
Could that be [ArtRepair](https://cibsr.stanford.edu/tools/human-brain-project/artrepair-software.html).